conda-recipe/conda_build_config.yaml

```diff
@@ -1,6 +1,12 @@
+c_stdlib:
+- sysroot # [linux]
+- macosx_deployment_target # [osx]
+c_stdlib_version:
+- '2.12' # [linux]
+- '11.0' # [osx]
+numpy:
+- '1.22'
 python:
-  - 3.8
-  - 3.9
-  - 3.10
-CONDA_BUILD_SYSROOT:
-  - /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk  # [osx]
\ No newline at end of file
+- '3.8'
+- '3.9'
+- '3.10'
```

Comment:
Is this related to the fix, or just a driveby?

Reply:
just to make things more uniform